docs/changes.rst

```diff
@@ -37,6 +37,20 @@ Released: not yet
   surrounded by verbose test (if p.parse.verbose:). See pywbemcli issue
   #395,
 
+* Fixes bugs in SubscriptionManager where 1. host_name  of
```

Comment:
Do not abbreviate the class name, i.e. use WBEMSubscriptionManager.

Reply:
DONE